### PR TITLE
fix: build converted guid to string properly

### PR DESCRIPTION
**Pull request by Helias**
Closes https://github.com/azerothcore/mod-npc-talent-template/issues/33

### CODE REVIEW

File: src/TemplateNPC.cpp [hunk 1]
```diff
@@ -673,7 +673,7 @@ void sTemplateNPC::ExtractGearTemplateToDB(Player* player, std::string& playerSp
 void sTemplateNPC::ExtractTalentTemplateToDB(Player* player, std::string& playerSpecStr)
 {
     QueryResult result = CharacterDatabase.PQuery("SELECT spell FROM character_talent WHERE guid = '%u' "
-        "AND talentGroup = '%u';", player->GetGUID(), player->GetActiveSpecMask());
+        "AND talentGroup = '%u';", (player->GetGUID()).GetRawValue(), player->GetActiveSpecMask());
```
Comment (UltraNix) [inline src/TemplateNPC.cpp:676]: `GetCounter`, not `GetRawValue`

File: src/TemplateNPC.cpp [hunk 2]
```diff
@@ -700,7 +700,7 @@ void sTemplateNPC::ExtractTalentTemplateToDB(Player* player, std::string& player
 void sTemplateNPC::ExtractGlyphsTemplateToDB(Player* player, std::string& playerSpecStr)
 {
     QueryResult result = CharacterDatabase.PQuery("SELECT glyph1, glyph2, glyph3, glyph4, glyph5, glyph6 "
-        "FROM character_glyphs WHERE guid = '%u' AND talentGroup = '%u';", player->GetGUID(), player->GetActiveSpec());
+        "FROM character_glyphs WHERE guid = '%u' AND talentGroup = '%u';", player->GetGUID().ToString().c_str(), player->GetActiveSpec());
```
Comment (UltraNix) [inline src/TemplateNPC.cpp:703]: `GetCounter` instead of string.

Comment (Helias) [reply src/TemplateNPC.cpp:703]: I was thinking about GetCounter, but I was not sure :sweat_smile: 
thanks!

**Comment by Helias** [comment]
improved, thanks @UltraNix

**Pull request merged**